roadmap/implementers-guide/src/node/approval/approval-distribution.md

```diff
@@ -40,14 +40,25 @@ struct State {
   finalized_number: BlockNumber,
 }
 
+enum MessageFingerprint {
+  Assigment(Hash, u32, ValidatorIndex),
+  Approval(Hash, u32, ValidatorIndex),
```

Comment:
What do the integers mean here?

Reply:
Unclear.  We add a received timestame to assignments when verifying their VRF signature.  Approval votes might benefit form a received timestame too.